doc/source/getting_started/why_xorbits/comparisons.rst

```diff
@@ -0,0 +1,202 @@
+.. _xorbits_vs_dask_modin_koalas:
+
+Xorbits Pandas vs. DaskDF vs. Modin vs. Koalas
+==============================================
+
+Libraries such as `Dask DataFrame`_ (short for DaskDF), `Pandas API on Spark`_ (short for Koalas), 
+and `Modin`_ aim to support the pandas API on top of distributed computing frameworks. Externally,
+these libraries mimic the pandas API. But the aims of these libraries are fundamentally different. 
```

Comment:
These 2 sentences are a little bit confusing, are you meaning "All of these libs has similar pandas-like API, but their implementation is quite different."?

```suggestion
and `Modin`_ aim to support the pandas API on top of distributed computing frameworks. Externally,
these libraries mimic the pandas API. But the aims of these libraries are fundamentally different. 
```

Reply:
Indeed, the difference in implementation results from the aim/design goal. If this is true, how can I enhance the readability here?